sdk/translation/Azure.AI.Translation.Document/samples/Sample2_PollIndividualDocuments.md

```diff
@@ -27,25 +27,24 @@ DocumentTranslationOperation operation = await client.StartTranslationAsync(inpu
 
 TimeSpan pollingInterval = new(1000);
 
-AsyncPageable<DocumentStatusResult> documents = operation.GetAllDocumentStatusesAsync();
-await foreach (DocumentStatusResult document in documents)
+await foreach (DocumentStatusResult document in operation.GetAllDocumentStatusesAsync())
 {
-    Console.WriteLine($"Polling Status for document{document.TranslatedDocumentUri}");
+    Console.WriteLine($"Polling Status for document{document.SourceDocumentUri}");
```

Comment:
`TranslatedDocumentUri` is only available when the document succeeds, so using `SourceDocumentUri` which is always available